cpp/src/column/column_factories.cpp

```diff
@@ -81,4 +85,58 @@ std::unique_ptr<column> make_fixed_width_column(
   return make_numeric_column(type, size, state, stream, mr);  
 }
 
+struct column_from_scalar_dispatch 
+{
+  template <typename T>
+  std::enable_if_t<cudf::is_fixed_width<T>(), std::unique_ptr<cudf::column>>
+  operator()( scalar const& value, size_type size,
+              rmm::mr::device_memory_resource* mr, cudaStream_t stream)
+  {
+    if( !value.is_valid() )
+      return make_fixed_width_column(value.type(), size, mask_state::ALL_NULL, stream);
+    auto output_column = make_fixed_width_column(value.type(),size,mask_state::UNALLOCATED,stream,mr);
+    auto view = output_column->mutable_view();
+    experimental::detail::fill_in_place(view, 0, size, value, stream);
+    return output_column;
+  }
+
+  template <typename T>
+  std::enable_if_t<std::is_same<cudf::string_view, T>::value, std::unique_ptr<cudf::column>>
+  operator()( scalar const& value, size_type size,
```

Comment:
None of the `operator()`'s seem to mutate any state (the class they belong to doesn't have any), in which case they should be `const`: `operator()(...) const`, etc.